src/arize_otel/_register.py

```diff
@@ -167,6 +167,7 @@ def create_resource(
     if model_version:
         attributes["model_version"] = model_version
     if project_name:
+        attributes["model_id"] = project_name
         attributes[ResourceAttributes.PROJECT_NAME] = project_name
```

Comment:
Let's file ticket to make receiver understand `ResourceAttributes.PROJECT_NAME` and translate it as model_id sever side?